policy/README.md

```diff
@@ -0,0 +1,353 @@
+
+References to geography (areas, street segments, etc.) will be done via UUID.  Geographic data will be available as GeoJSON via the `/geographies` endpoint.  In a future revision of Agency, we will reconcile this with the existing `/service_areas` endpoint.  `/service_areas` currently only handles GeoJSON MultiPolygon and Polygon objects, and Policies might refer to street segments or points.  
+
+This initial draft proposes a subset of possible policies for consideration, and should not be taken to be the a comprehensive enumeration of all possible policies.
+
+<a name="distribution"></a>
+## Distribution
+
+Policies may be published by Agencies or their authorized delegates via JSON objects via REST API.
+
+Each policy will have a unique ID (UUID).
+
+Published policies should be treated as immutable data.  Obsoleting or otherwise changing policies is accomplished by publishing additional policies with a field named `prev_policies`, a list of UUID references to the previous policy or policies.  
+
+Policies should be stored and accessible indefinitely so that the set of active policies at a given time in the past can be retrieved from the `/policies` endpoint.
+
+Policies will typically be linked to one or more associated geographies.  Geography descriptions (e.g. geofences or lists of street segments) must also be maintained by the Agency indefinitely.  Policies without specific geographies (global policies) are assumed to apply to the entire service area managed by the Agency.
+
+Policies should be re-fetched whenever (a) a Policy expires (via its `end_date`), or (b) at an interval specified by the Agency, e.g. "daily at midnight".
+
+
+<a name="schema"></a>
+## Schema
+
+<a name="policy-fields"></a>
+### Policy Fields
+
+| Name            | Type      | R/O | Description |
+| ---             | ---       | --- | --- |
+| `name`          | String    | R   | Name of policy |
+| `policy_id`     | UUID      | R   | Unique ID of policy |
+| `description`   | String    | R   | Description of policy |
+| `start_date`    | timestamp | R   | Beginning date/time of policy enforcement |
+| `end_date`      | timestamp | O   | End date/time of policy enforcement |
+| `prev_policies` | UUID[]    | O   | Unique IDs of prior policies replaced by this one |
+| `rules`         | Rule[]    | R   | List of applicable rule elements (see [“rule fields”](#rule-fields)) |
+
+Note that the provider-scope is at present not included.  A Provider should not be able to tell whether the policy is specific to them, or to a subset of Provider, or all Providers.
+
+<a name="rule-types"></a>
+### Rule Types 
+
+| Name    | Description |
+| ---     | --- |
+| `count` | Fleet counts based on regions.  Rule max/min refers to number of devices. |
+| `time`  | Individual limitations on time spent in one or more vehicle-states.  Rule max/min refers to number of minutes. |
+| `speed` | Global or local speed limits.  Rule max/min refers to miles per hour. |
```

Comment:
mph or should we allow different units here. I'm not an i18n expert but we should probably do some research on how best to handle this.

Reply:
Units are specified in the `Rule` object.  `mph` or `kph` for `speed` rules.